### DISCUSSION TITLE
Record only at specific time

### DISCUSSION THREAD

**Discussion by Code-TT** [blakeblackshear/frigate#5851] [General]
I use Frigate to monitor the enclosure of our chickens. Actually, I only need a recording overnight (because of rats, maggots, foxes or other intruders) because during the day there is always movement and the hard drive fills up too quickly for me. How to set the time control?

**Comment by NickM-27** [comment]
You can use the mqtt API along with some automation software (NodeRED, Home Assistant, etc.) to automate when recording is enabled

**Reply by Code-TT** [reply]
Ahh, ok :)
And how exactly? Just set the State from ON to OFF? Which one? All of them?

![Screenshot 2023-03-27 225928](https://user-images.githubusercontent.com/37022171/228065356-4995d76b-a396-4f20-86fa-0bc10e905d10.png)

**Reply by NickM-27** [reply]
write to https://docs.frigate.video/integrations/mqtt#frigatecamera_namerecordingsset

**Reply by Code-TT** [reply]
It works! Thank you! :)

**Reply by PAlvesLancs** [reply]
Hi @Code-TT can you please show me how you did? I am a noobie on this and I am a bit stuck.

Thanks a million
